### ISSUE TITLE
[Arb] Executed transaction on-chain still shows in the pending transaction

### ISSUE THREAD

**Issue by Sherie15** [5afe/safe-support#301]
### Issue Category

Transaction missing / not visible in app or service

### What happened?

The user already executed nonce 17 on-chain but still showing in the transaction pending tab and nonce 18 is not showing in the pending transaction queue.

![image](https://github.com/5afe/safe-support/assets/113986185/13893773-e1b3-448d-9aed-d3801a35a0af)

https://arbiscan.io/tx/0x1f61f8bdf07c6de9f37e619b064fb7f18571a9cb4c42bc2d3b2f9b31274664e6

Another user is experiencing the same problem replacement for nonce 103 is not showing the pending transaction queue: 
0x06b5d8738D4b2A5374f6a5fAdC5117EeB123829b

![image](https://github.com/5afe/safe-support/assets/113986185/c24e6013-8fb9-45b3-8367-2eeba9ad85a8)

### Date and time this happened or you first noticed this issue

10-10-2023 08:49am

### Network

Arbitrum

### Token Type

None

**Comment by 0xganz** [comment]
+1

**Comment by JagoFigueroa** [comment]
Hola team! Issue should be resolved now 😄

**Comment by JagoFigueroa** [comment]
You are right @CindyLo0 something is still off 😢 The team is investigating, we will get back to you as soon as we know more 🙏

**Comment by rkhadder** [comment]
+1 still experiencing this issue for our arb1 safes
